doc/calendar.md

```diff
@@ -1,3 +1,13 @@
 # Calendar
 
 Upcoming events...
+
+CESM Ocean Working Group and MOM6 Developers & Users Meeting:
+18-19 March 2020 at NCAR Mesa Lab, Boulder, CO.
+http://www.cesm.ucar.edu/working_groups/Ocean
+There will be many CPT presenters and CPT-related discussions and online streaming of the event.
```

Comment:
Might be nice to include some formatting in here. For example, the following will produce nice nested headings.

```markdown
## Upcoming Events

### CESM Ocean Working Group
...

## Past Events

### Ocean Sciences Meeting 2020
...
```

```